runtime/early/extra-mappings.vim

```diff
@@ -0,0 +1,9 @@
+" Extra default keymaps and keyboard behavior
+" see :help nvim-extra-mappings
+
+tnoremap <esc><esc> <C-\><c-n>
+nnoremap Y y$
```

Comment:
I don't see this being very useful in visual mode. Didn't someone mention this mapping should use `nnoremap` instead of `noremap` in the other PR?